homeassistant/components/jellyfin/media_source.py

```diff
@@ -529,7 +534,13 @@ def _get_stream_url(self, media_item: dict[str, Any]) -> str:
         item_id = media_item[ITEM_KEY_ID]
 
         if media_type == MEDIA_TYPE_AUDIO:
-            return self.api.audio_url(item_id)  # type: ignore[no-any-return]
+            if self.entry.options.get("audio_codec") in SUPPORTED_AUDIO_CODECS:
```

Comment:
I would suggest to make a constant for `audio_codec`. Maybe `CONF_AUDIO_CODEC`?

```suggestion
            if audio_codec := self.entry.options.get("CONF_AUDIO_CODEC"):
```

Schema ensures that is a valid one